pkg/controller/operators/olm/operator.go

```diff
@@ -345,6 +346,7 @@ func (a *Operator) checkReplacementsAndUpdateStatus(csv *v1alpha1.ClusterService
 		log.Infof("newer ClusterServiceVersion replacing %s, no-op", csv.SelfLink)
 		msg := fmt.Sprintf("being replaced by csv: %s", replacement.SelfLink)
 		csv.SetPhase(v1alpha1.CSVPhaseReplacing, v1alpha1.CSVReasonBeingReplaced, msg)
+		metrics.CSVUpgradeCount.Inc()
```

Comment:
I wonder if we even need this if we have the other metrics. `count(changes(csv_count{phase=Replacing} / 2))` might get us there?  

Though with a counter we'll eventually get an accurate count even if we can't scrape some of the events...

Reply:
I left it for now, but I'd be happy to remove since it doesn't really fit in with the theme of the rest of them.